tests/Agent/IntegrationTests/UnboundedIntegrationTests/MySql/MySqlAsyncTests.cs

```diff
@@ -125,4 +125,22 @@ public void Test()
             );
         }
     }
+
+    [NetFrameworkTest]
```

Comment:
We should add additional subclasses to test the other versions.  For example:

`public class MySqlAsyncTestsFW471 : MySqlAsyncTestsBase<ConsoleDynamicMethodFixtureFW471>`

etc. (so four more subclassed versions of the tests - two for Framework (471 and 462) and two for core (coreapp3.1 and net5.0)